RxStarscreamTests/RxStarscreamTests.swift

```diff
@@ -19,7 +19,7 @@ public func ==(lhs: WebSocketEvent, rhs: WebSocketEvent) -> Bool {
     case (.connected, .connected):
         return true
     case (.disconnected(let lhsError), .disconnected(let rhsError)):
-        return lhsError == rhsError
+        return lhsError?.localizedDescription == rhsError?.localizedDescription
```

Comment:
I'm not sure if testing description equality is proper here. Didn't the previous `lhsErr == rhsErr` work for you ?

Reply:
Have tried but an error was popping when tried to build. Previously I had a comment on this with "TO REVIEW". So I'm a bit lost on this one, do you know if Error suffered from any change with the swift 4 update?

Reply:
I'll take a poke at this :) Maybe it's a custom error that doesn't conform to Equatable. I'll look.

Reply:
From what I've seen it's not a custom error, but I could be wrong.